package.json

```diff
@@ -2,10 +2,20 @@
   "name": "koa-body",
   "version": "5.0.0",
   "description": "A Koa body parser middleware. Supports multipart, urlencoded and JSON request bodies.",
-  "main": "index.js",
-  "types": "./index.d.ts",
+  "main": "lib/index.js",
+  "files": [
+    "lib",
+    "LICENSE",
+    "README.md",
+    "package.json"
+  ],
   "scripts": {
-    "test": "mocha test/unit/",
+    "build": "npm run check-format && npm run test && npm run clean && npm run build:typescript",
+    "build:typescript": "tsc --build tsconfig.build.json",
```

Comment:
Can we add  `"prepack": "npm run build",` to ensure we only publish with built and tested code?

Reply:
done